src/main/java/sirius/biz/scripting/CustomEvent.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.scripting;
+
+import sirius.kernel.health.HandledException;
+
+import java.util.Optional;
+
+/**
+ * Provides a base class for all custom events handled by a {@link CustomEventDispatcher}.
+ */
+public abstract class CustomEvent {
+
+    protected boolean success;
+    protected boolean failed;
+    protected HandledException error;
```

Comment:
**Optional:** Would like the JavaDoc of the methods on the fields themselves, as this helps when implementing or debugging subclasses of this abstract class.

Reply:
done